src/Algebra/Solver/Monoid/Tactic.agda

```diff
@@ -0,0 +1,52 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- A solver for equations over monoids
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Algebra.Bundles using (Monoid)
+import Algebra.Solver.Monoid.Expression as Expression
+
+module Algebra.Solver.Monoid.Tactic {a c ℓ} (M : Monoid c ℓ)
+  (open Monoid M using (rawMonoid; setoid; _≈_))
+  (open Expression rawMonoid using (Expr; Env; var; ⟦_⟧; NormalAPI))
+  (N : NormalAPI {a}) where
+
+open import Data.Maybe.Base as Maybe
+  using (Maybe; From-just; from-just)
+open import Data.Nat.Base using (ℕ)
+open import Function.Base using (_∘_; _$_)
+open import Relation.Binary.Consequences using (dec⇒weaklyDec)
+open import Relation.Binary.PropositionalEquality.Core using (_≡_; cong)
+import Relation.Binary.Reflection as Reflection
+
+open NormalAPI N
+open module R = Reflection setoid var ⟦_⟧ (⟦_⟧⇓ ∘ normalise) correct public
+  using (solve; _⊜_)
+
+private
+  variable
+    n : ℕ
+
+
```

Comment:
One too many empty lines?

Reply:
OK, I thought the `style-guide` recommended two after the initial `private` block`, but maybe that's been superseded now...

Reply:
Oh sorry, I haven't seen that entry in the style guide before! That's a new one to me, apologies.